config/currency_iso.json

```diff
@@ -1584,6 +1584,7 @@
     "iso_code": "NIO",
     "name": "Nicaraguan Córdoba",
     "symbol": "C$",
+    "disambiguate_symbol": "NIO$",
```

Comment:
Is this actually being used anywhere? I've tried googling, but unfortunately found no results…